hack/common.sh

```diff
@@ -14,25 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Copyright 2022 The Kepler Contributors
+# Copyright 2023 The Kepler Contributors
```

Comment:
We don't need to update the copyright date. Actually should should be the date that the file was created.

Reply:
Oh, just know about this, let me fix later.